packages/edit-site/src/components/page-pages/index.js

```diff
@@ -307,6 +324,21 @@ export default function PagePages() {
 					isLoading={ isLoadingPages || isLoadingAuthors }
 					view={ view }
 					onChangeView={ onChangeView }
+					selection={ selection }
+					setSelection={ setSelection }
+					getSelectionLabel={ ( isSelected, item ) =>
```

Comment:
I think we should probably pass a `labels` object([example](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/store/actions.js#L1811)) and handle this inside DataViews. For example we also need at least a global search label and we will probably need more in the future.

Reply:
Agree. I'd rather go with the generic approach for now (select item/deselect item) and don't have this prop.